charts/kserve-resources/values.yaml

```diff
@@ -424,5 +424,8 @@ kserve:
         image: kserve/kserve-localmodelnode-agent
         tag: *defaultVersion
         reconcilationFrequencyInSecs: 60
+        securityContext:
+          runAsUser: 0
+          runAsNonRoot: false
```

Comment:
Looks like this is required, otherwise it would complain running as root is not allowed.